ontology/observable/observable.ttl

```diff
@@ -5440,22 +5440,22 @@ observable:SoftwareFacet
 	rdfs:comment "A software facet is a grouping of characteristics unique to a software program (a definitively scoped instance of a collection of data or computer instructions that tell the computer how to work). [based on https://en.wikipedia.org/wiki/Software]"@en ;
 	sh:property
 		[
-			sh:datatype xsd:string ;
+			sh:class core:IdentityAbstraction ;
```

Comment:
core:IdentityAbstraction should only be used within the core namespace. It is a mechanism to allow references to Identity within core without causing cyclical importing.
In the observable namespace, uses such as this should directly use identity:Identity.

Reply:
Resolved in d7a593e.